libs/core-ui/src/lib/util/CohortUtils.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { JointDataset } from "./JointDataset";
+
+enum ColumnNames {
+  Index = "Index",
```

Comment:
Should these come from the english json file directly?  Instead of redefining here we can just use the existing english definitions (so it's easier to update in case we decide to change the strings and also to limit duplication).

Reply:
makes sense, how to use the english json file directly?
this is a temporary fix, was planning to change the filter logic with the removing jointdataset work, but I am happy to try other ways now

Reply:
umm I do not recommend using from json file. Haven't seen en-us file being used other than for localization.

Reply:
yeah, agree on having single source of truth, but not sure if we can use directly from json file

Reply:
Can we reuse CohortColumnNames from ProcessPreBuiltCohort.ts file?

Reply:
Or we can fix Jointdataset itself, but I don't want to touch too much on that, as it is going to be gone. So made this temporary fix

Reply:
> Can we reuse CohortColumnNames from ProcessPreBuiltCohort.ts file?

RegressionError is different, we are passing "Regression error" as filter column to backend instead of "Error"(CohortColumnNames.RegressionError)

Reply:
hmm ok then you can just ignore my comment!  Seems like this is a temporary fix anyway.